README.md

```diff
@@ -52,10 +59,22 @@ source openembedded-core/oe-init-build-env
 ../mc_build.sh -m <machine> -c <container config>:<container image> -h <host config>:<host_image> -d container_image
 ```
 
-To do a "dry run" without running a build, add `-e` which emits what would have run if you ran this from bitbake.
-If you want to disable the autostarting of the container, use -d with a comma delineated list of the container image name.
+To do a "dry run" without running a build, add `-e` which emits what would have
+run if you ran this from bitbake.
+If you want to disable the autostarting of the container, use -d with a comma
+delineated list of the container image name.
+
+If you are familiar with Yocto Project development and multiconfig and wish to
+use bitbake directly, see the local.conf for variables that need to be set. For
+running bitbake commands, including outside of building, such as
+`bitbake-layers show-layers`, run `../mc_build.sh -e -v <VENDOR>` from the build
```

Comment:
`../mc_build.sh -e -v <VENDOR>` also requires the ` -m <MACHINE>` to be specified.